tensorflow/core/lib/random/random_distributions.h

```diff
@@ -95,47 +133,57 @@ template <class Generator>
 class UniformDistribution<Generator, bfloat16> {
  public:
   // The number of elements that will be returned.
-  static constexpr int kResultElementCount = Generator::kResultElementCount;
+  // Set the number to be greater equal to Eigen packet size of type,
+  // so computations can be vectorized using SIMD.
+  static constexpr int kVectorLength =
+      Eigen::internal::packet_traits<bfloat16>::size;
+  static constexpr int kResultElementCount =
+      (kVectorLength > Generator::kResultElementCount)
+          ? kVectorLength
+          : Generator::kResultElementCount;
```

Comment:
I think this is easier to read:
```suggestion
  static constexpr int kResultElementCount =
      std::max(kVectorLength, Generator::kResultElementCount));
```
Same for the class for `float`.

Reply:
Done.